sources/CascadiaCode-Bold.ufo/fontinfo.plist

```diff
@@ -204,7 +204,7 @@
     <key>openTypeNameLicense</key>
     <string>Microsoft supplied font. You may use this font to create, display, and print content as permitted by the license terms or terms of use, of the Microsoft product, service, or content in which this font was included. You may only (i) embed this font in content as permitted by the embedding restrictions included in this font; and (ii) temporarily download this font to a printer or other output device to help print content. Any other use is prohibited.
  
-The following license, based on the SIL Open Font license (https://scripts.sil.org/OFL), applies to this font. Additional license terms may be found on the GitHub repository for this font (https://github.com/microsoft/cascadia-code/blob/main/LICENSE).
+The following license, based on the SIL Open Font license (https://scripts.sil.org/OFL), applies to this font. Additional license terms may be found on the GitHub repository for this font (https://github.com/microsoft/cascadia-code/blob/master/LICENSE).
```

Comment:
This merge is incorrect -- it changed the URL back from `main` to `master`!

Reply:
Oh fun. Must have been an old version on my other computer that snuck in.